port/linux/ipadapter.c

```diff
@@ -1529,6 +1529,14 @@ void
 oc_connectivity_shutdown(size_t device)
 {
   ip_context_t *dev = oc_get_ip_context_for_device(device);
+
+#ifdef OC_HAS_FEATURE_BRIDGE
```

Comment:
remove the ifdef, lets keep this defensive check globally

Reply:
done